<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1112</width>
    <height>790</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="mainWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>450</y>
      <width>581</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="sectionLayout"/>
   </widget>
   <widget class="QLabel" name="imageDisplay">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>581</width>
      <height>401</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="opticalParams" native="true">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>450</y>
      <width>421</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="positionParams" native="true">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>30</y>
      <width>141</width>
      <height>411</height>
     </rect>
    </property>
    <widget class="QLabel" name="positionLabel">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Position</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSlider" name="positionSlider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>22</width>
       <height>401</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="positionValue">
     <property name="geometry">
      <rect>
       <x>76</x>
       <y>83</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>PositionVlaue</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="meanParams" native="true">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>30</y>
      <width>421</width>
      <height>321</height>
     </rect>
    </property>
    <widget class="QSlider" name="meanSlider">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>22</width>
       <height>81</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="meanLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Moyenne</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="meanValue">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Moyenne</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
